src/bptl/tasks/migrations/0010_auto_20201207_1818.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 2.2.14 on 2020-12-07 17:18
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("tasks", "0009_auto_20201207_1814"),
+    ]
+
+    operations = [
+        migrations.AlterModelTable(
+            name="defaultservice",
+            table=None,
```

Comment:
yeah - this was some magic to move the model from one app to another. `bptl.work_units.zgw.migrations.0004` explicitly sets the table name to `tasks_defaultservice`.

This migration basically un-does that explicit table name specifying, but the end result is the same because the model is in the app `tasks` now, and has the name `defaultservice`.

Reply:
Looks like black magic :)